### ISSUE TITLE
Training procedure questions for Hierarchical Story Generation

### ISSUE THREAD

**Issue by apappu97** [pytorch/fairseq#631]
Hi, working on the Hierarchical Story Generation repo -- trying to replicate training the original fusion seq2seq model using the pretrained frozen checkpoint. I was wondering if you could post details on training procedure, i.e. how many epochs/iterations the model was trained for, loss figures/numbers, etc, as it would be really helpful for benchmarking/reproducing. Thanks!

**Comment by huihuifan** [comment]
I trained for about 30 epochs, and the perplexity numbers are in the paper reported on this PyTorch version.
